<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="sys_ui_application">
    <sys_ui_application action="INSERT_OR_UPDATE">
        <active>true</active>
        <hint/>
        <name>Sample Exception</name>
        <order/>
        <roles>x_354377_exception.sample_exception_user</roles>
        <sys_class_name>sys_ui_application</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2019-06-25 08:32:32</sys_created_on>
        <sys_domain>global</sys_domain>
        <sys_domain_path>/</sys_domain_path>
        <sys_id>a709fe70db1a7700f15d5c7a489619af</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>Sample Exception</sys_name>
        <sys_overrides/>
        <sys_package display_value="Exception" source="x_354377_exception">baa83630db1a7700f15d5c7a48961994</sys_package>
        <sys_policy/>
        <sys_scope display_value="Exception">baa83630db1a7700f15d5c7a48961994</sys_scope>
        <sys_update_name>sys_ui_application_a709fe70db1a7700f15d5c7a489619af</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2019-06-25 08:32:32</sys_updated_on>
    </sys_ui_application>
</record_update>
